explainer.md

```diff
@@ -0,0 +1,329 @@
+because the APIs aren't yet sufficient.
+New web platform features aren't fully accessible
+or don't interact well with existing accessibility APIs,
+forcing developers to choose between using new standard or remaining accessible.
+
+* A library author creating a custom element is forced to "sprout" ARIA attributes
+to express semantics which are implicit for native elements.
+
+```html
+<!-- Page author uses the custom element as they would a native element -->
+<custom-slider min="0" max="5" value="3"></custom-slider>
+
+<!-- Custom element is forced to "sprout" extra attributes to express semantics -->
+<custom-slider min="0" max="5" value="3" role="slider"
+               tabindex="0" aria-valuemin="0" aria-valuemax="5"
+               aria-valuenow="3" aria-valuetext="3"></custom-slider>
+```
+
+* Moreover, there is no way to connect custom HTML elements to accessible actions.
+For example, the custom slider above with a role of `slider`
+prompts a suggestion on VoiceOver for iOS
+to perform swipe gestures to increment or decrement,
+but there is no way to handle that gesture via the DOM API.
+
+* Many ARIA relationship properties depend on IDREFs,
+meaning that elements participating in these relationship must have globally unique IDs.
+This is ugly, costly and fiddly to achieve,
+particularly in the context of framework or library code.
+
+* A custom element which uses shadow DOM
+and which needs to express semantic relationships such as
+["active descendant"](https://www.w3.org/TR/wai-aria-1.1/#aria-activedescendant)
+may be unable to do so, as `aria-activedescendant` relies on IDREF values,
+which are scoped to a single document fragment.
+
+```html
+<x-combobox>
+  #shadow-root
+  |  <!-- this doesn't work! -->
+  |  <input aria-activedescendant="opt1"></input>
+  <x-optionlist>
+    <x-option id="opt1">Option 1</x-option>
+    <x-option id="opt2">Option 2</x-option>
+    <x-option id='opt3'>Option 3</x-option>
+ </x-optionlist>
+</x-combobox>
+
+```
+
+
+* Custom UI which uses canvas to draw the UI currently 
+
+A low-level API would bridge these gap,
+allowing authors to bypass artificial limitations or bugs in the platform
+and provide a custom accessible experience where necessary.
+
+### Audience for the proposed API
+
+We don't expect or encourage the average web developer
+to want or need a low-level accessibility API for most things.
+Declarative HTML should be sufficient for most web page authors.
+
+This API is aimed at the relatively small number of developers who create and maintain
+the JavaScript frameworks and widget libraries that power the vast majority of web apps.
+Accessibility is a key goal of most of these frameworks and libraries,
+as they need to be usable in as broad a variety of contexts as possible.
+A low-level API would allow them to work around bugs and limitations
+and provide a clean high-level interface that "just works" for the developers who use their components.
```

Comment:
This might be a good area to drop in mention of the extensible web manifesto for bonus points. Being able to explain how accessibility works and give developers an even playing field seems like a good thing